### PR TITLE
Rephrase observations' definition in Lunar Lander Environment

### PR DESCRIPTION
**Pull request by EvanMath**
# Description

Rephrase a part of Lunar Lander docstrings. The observation definition was misleading.

Fixes # (issue)
> **There are 8 states**: the coordinates of the lander in x & y, its linear velocities in x & y, its angle, its angular velocity, and two booleans that represent whether each leg is in contact with the ground or not. 

Changed to:
> States are eight (8) dimensional vector: the coordinates of the lander in `x` & `y`, its linear velocities in `x` & `y`, its angle, its angular velocity, and two booleans that represent whether each leg is in contact with the ground or not.

## Type of change
Docstring Change

Please delete options that are not relevant.

- [ ] This change requires a documentation update

### Screenshots
Please attach before and after screenshots of the change if applicable.

# Checklist:

- [ ] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: gym/envs/box2d/lunar_lander.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ class LunarLander(gym.Env, EzPickle):
     orientation engine, fire main engine, fire right orientation engine.
 
     ### Observation Space
-    There are 8 states: the coordinates of the lander in `x` & `y`, its linear
+    States are eight (8) dimensional vector: the coordinates of the lander in `x` & `y`, its linear
```
Comment (RedTachyon) [inline gym/envs/box2d/lunar_lander.py:102]: I'd rephrase this to "The state is an 8-dimensional vector"

**Comment by pseudo-rnd-thoughts** [comment]
LGTM
